libs/payments/ui/src/lib/server/components/UpgradePurchaseDetails/en.ftl

```diff
@@ -0,0 +1,14 @@
+## Component - UpdatedPurchaseDetails
+
+upgrade-purchase-details-current-plan-label = Current plan
+upgrade-purchase-details-new-plan-label = New plan
+upgrade-purchase-details-promo-code = Promo Code
+upgrade-purchase-details-tax-label = Taxes and Fees
+upgrade-purchase-details-new-total-label = New total
+upgrade-purchase-details-prorated-upgrade = Prorated Upgrade
+
+# $productName (String) - Name of the upgraded product (e.g. Mozilla VPN)
```

Comment:
This comment either needs to be a group comment, or repeated for each string. Otherwise this comment will only appear for the first string below (`-daily`) but none of the others (`-weekly` etc.).

Also as I look at this again, the context about what (Daily) is referring to might not be clear- could you indicate in a comment what Daily / Weekly/ Monthly / Yearly is referring to?